### PR TITLE
Symfony 5.4 Docs Update

### PR DESCRIPTION
**Pull request by Phocacius**
PR that adds relevant Symfony 5.4 information to Mapbender docs, source is [this Mapbender starter PR]( https://github.com/mapbender/mapbender-starter/pull/110).

### CODE REVIEW

File: de/customization/templates.rst [hunk 1]
```diff
@@ -263,12 +263,12 @@ Bei unveränderter Übernahme der Stylevorgaben sieht die Administration dann so
 
 
 
-Registrieren des Bundles in app/AppKernel.php
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Registrieren des Bundles in scr/Kernel.php
```
Comment (Phocacius) [inline de/customization/templates.rst:266]: This is not true anymore, new bundles should now be registered in `config/bundles.php`. That file looks like this:

```php
<?php

return [
    Symfony\Bundle\AclBundle\AclBundle::class => ['all' => true],
    Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true],
    Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
    Symfony\Bundle\MonologBundle\MonologBundle::class => ['all' => true],
    Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
    Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
    Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
    Mapbender\CoreBundle\MapbenderCoreBundle::class => ['all' => true],
    Mapbender\FrameworkBundle\MapbenderFrameworkBundle::class => ['all' => true],
    FOM\CoreBundle\FOMCoreBundle::class => ['all' => true],
    FOM\ManagerBundle\FOMManagerBundle::class => ['all' => true],
    FOM\UserBundle\FOMUserBundle::class => ['all' => true],
    OwsProxy3\CoreBundle\OwsProxy3CoreBundle::class => ['all' => true],
    Mapbender\WmcBundle\MapbenderWmcBundle::class => ['all' => true],
    Mapbender\WmsBundle\MapbenderWmsBundle::class => ['all' => true],
    Mapbender\WmtsBundle\MapbenderWmtsBundle::class => ['all' => true],
    Mapbender\ManagerBundle\MapbenderManagerBundle::class => ['all' => true],
    Mapbender\PrintBundle\MapbenderPrintBundle::class => ['all' => true],
    Mapbender\MobileBundle\MapbenderMobileBundle::class => ['all' => true],
    Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle::class => ['all' => true],
];
```

To add a bundle, add a line with the fully qualified class name of your bundle file. `['all' => true]` means the bundle is active in all configurations. `['dev' => true]` means, the bundle is only active in the development environment.

File: de/customization/templates.rst [hunk 2]
```diff
@@ -302,9 +302,9 @@ Aktualisieren Sie das web-Verzeichnis. Jedes Bundle hat seine eigenen Assets - C
 
 .. code-block:: bash
 
-    app/console assets:install web
+    bin/console assets:install web
```
Comment (Phocacius) [inline de/customization/templates.rst:305]: public

File: de/customization/templates.rst [hunk 3]
```diff
@@ -319,7 +319,7 @@ Die neue Anwendungs-Vorlage kann über verschiedene Wege verwendet werden:
 Einbindung in YAML-Anwendungen
 ******************************
 
-Sie können nun die YAML-Anwendungen unter ``app/config/applications`` anpassen und auf das neue Template verweisen.
+Sie können nun die YAML-Anwendungen unter ``application/config/applications`` anpassen und auf das neue Template verweisen.
```
Comment (Phocacius) [inline de/customization/templates.rst:322]: without application

File: de/development/element_generate.rst [hunk 4]
```diff
@@ -115,7 +115,7 @@ Es wird eine Übersicht über die erfolgte Aktion ausgegeben. Es wurde eine PHP-
  - Your element WorkshopDemoBundle\Element\MapKlick has been created.
  - The following files have been created:
   - PHP class (src/Workshop/DemoBundle/Element/MapKlick.php)
-  - jQuery widget (src/Workshop/DemoBundle/Resources/public/mapbender.element.mapklick.js)
+  - jQuery widget (src/Workshop/DemoBundle/public/mapbender.element.mapklick.js)
```
Comment (Phocacius) [inline de/development/element_generate.rst:118]: Resources was correct

File: de/installation/installation_configuration.rst [hunk 5]
```diff
@@ -64,14 +64,14 @@ Jedes Bundle hat seine eigenen Abhängigkeiten - CSS-Dateien, JavaScript-Dateien
 
 .. code-block:: yaml
 
-    app/console assets:install web
+    bin/console assets:install web
```
Comment (Phocacius) [inline de/installation/installation_configuration.rst:67]: public

File: de/installation/installation_configuration.rst [hunk 6]
```diff
@@ -64,14 +64,14 @@ Jedes Bundle hat seine eigenen Abhängigkeiten - CSS-Dateien, JavaScript-Dateien
...
 
 
 Sie können auch einen symbolischen Link verwenden, statt die Dateien zu kopieren.  Dies erleichtert die Bearbeitung der abhängigen Dateien in den bundle-Verzeichnissen.
 
 .. code-block:: yaml
 
-   app/console assets:install web --symlink --relative
+   bin/console assets:install web --symlink --relative
```
Comment (Phocacius) [inline de/installation/installation_configuration.rst:74]: public

File: de/installation/installation_symfony.rst [hunk 7]
```diff
@@ -18,7 +18,7 @@ In dieser Anleitung wird die im Installationspaket mitgelieferte SQLite-Datenban
 
 .. code-block:: bash
 
-    app/console server:run
+    bin/console server:run
```
Comment (Phocacius) [inline de/installation/installation_symfony.rst:21]: not true anymore

**Review by astroidex — APPROVED** [review]

**Pull request merged**